### ISSUE TITLE
Add support for Symfony 7

### ISSUE THREAD

**Issue by garak** [Harborn-digital/cookie-consent-bundle#128]
Please

**Comment by davidromani** [comment]
Indeed, is not working with Symfony 6.4 and PHP Attributes configuration too.

**Comment by davidromani** [comment]
@garak check this fork if you want, for me it's working with Symfony 6.4 (don't know if still working with 7.0) https://github.com/davidromani/harborn-cookie-consent-bundle/tree/master

**Comment by LeoMoshko** [comment]
any news?

**Comment by SimonChabrier** [comment]
I made this this fork this morning : [https://github.com/SimonChabrier/cookie-consent-bundle](https://github.com/SimonChabrier/cookie-consent-bundle) its ok for symfony 7.* and php 8.*

**Comment by doublej74** [comment]
I went for @SimonChabrier 's version. He has a good description added to the readme as well. I have been able to add it, but still need to figure out some exceptions, probably related to configuration. I'm using v7.2 with php 8.3.x. I'll keep adding info here when I get it working.

**Comment by RonRademaker** [comment]
We have discontinued the development and support of this bundle. We thank you for your input.
